EuclideanGeometry/Foundation/Construction/Triangle/IsocelesTriangle.lean

```diff
@@ -12,15 +25,56 @@ variable {P : Type _} [EuclideanPlane P]
 
 section IsocelesTriangles
 
+open Triangle
 def Triangle.IsIsoceles (tri : Triangle P) : Prop := tri.edge₂.length = tri.edge₃.length
 
-theorem is_isoceles_tri_iff_ang_eq_ang_of_nd_tri {tri_nd : Triangle_nd P} : tri_nd.1.IsIsoceles ↔ (tri_nd.angle₂.value= - tri_nd.angle₃.value) := sorry
+
+
+
```

Comment:
please delete these empty lines. Please use only one empty line to separate theorems